packages/analyzer/CHANGELOG.md

```diff
@@ -5,10 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
-* Removed non-essential files from published package, such as tests.
+<!-- ## Unreleased -->
 <!-- Add new, unreleased changes here. -->
 
+## [3.2.2] - 2018-01-08
```

Comment:
2019-01-10

Reply:
Thanks. This is going to happen a lot to me I think.